warehouse/query-core/src/main/java/datawave/query/iterator/logic/TermFrequencyExcerptIterator.java

```diff
@@ -34,38 +34,59 @@
  * for each document scanned.
  */
 public class TermFrequencyExcerptIterator implements SortedKeyValueIterator<Key,Value>, OptionDescriber {
-    private static final Logger log = Logger.getLogger(TermFrequencyExcerptIterator.class);
+    private static final Logger log = LoggerFactory.getLogger(TermFrequencyExcerptIterator.class);
     private static final Joiner joiner = Joiner.on(" ").skipNulls();
 
-    // The field name option
+    /** The field name option */
     public static final String FIELD_NAME = "field.name";
-    // The start offset option
+    /** The start offset option */
     public static final String START_OFFSET = "start.offset";
-    // The end offset option
+    /** The end offset option */
     public static final String END_OFFSET = "end.offset";
-    // the underlying source
-    protected SortedKeyValueIterator<Key,Value> source;
 
-    // the field name
+    /** the underlying source */
+    protected SortedKeyValueIterator<Key,Value> source;
+    /** the field name */
     protected String fieldName;
-    // the start offset (inclusive)
+    /** the start offset (inclusive) */
     protected int startOffset;
-    // the end offset (exclusive)
+    /** the end offset (exclusive) */
     protected int endOffset;
 
-    // The specified dt/uid column families
+    /** The specified dt/uid column families */
     protected SortedSet<String> columnFamilies;
-    // inclusive or exclusive dt/uid column families
+    /** inclusive or exclusive dt/uid column families */
     protected boolean inclusive;
 
-    // the underlying TF scan range
+    /** the underlying TF scan range */
     protected Range scanRange;
 
-    // the top key
+    /** the top key */
     protected Key tk;
-    // the top value
+    /** the top value */
     protected Value tv;
 
+    /**
+     * The size of half of the original range. <br>
+     * Used during trimming.
+     */
+    private float origHalfSize;
+    /** the list of hit terms */
+    protected ArrayList<String> hitTermsList;
+    /** the direction for the excerpt */
+    private String direction;
+    /**
+     * Whether we might need to trim down the excerpt to the requested size. <br>
+     * <br>
+     * Is false if this is the first time running the iterator and true otherwise.
+     */
+    private boolean trim;
+
+    private static final String BEFORE = "BEFORE";
+    private static final String AFTER = "AFTER";
+    private static final String BOTH = "BOTH";
+    private static final String XXXWESKIPPEDAWORDXXX = "XXXWESKIPPEDAWORDXXX";
```

Comment:
Suggesting a slight re-ordering of these fields so that related fields are close to each other in the code (this violates conventions with the goal of making the code more readable). Also an attempt to further define what these things mean. Please check that my descriptions are accurate. Also favoring a more compact comment style. Sprinkled in some `<br>`'s to make the auto-formatter happy.
```suggestion
    /** the list of hit terms: terms from the query that resulted in the current document being returned as a result */
    protected ArrayList<String> hitTermsList;
    /** the direction for the excerpt - controls which directions we build the excerpt from an originating hit term */
    private String direction;
    /** specifies that an excerpt is built using the terms prior to the hit term(s) */
    private static final String BEFORE = "BEFORE";
    /** specifies that an excerpt is built using the terms after the hit term(s) */
    private static final String AFTER = "AFTER";
    /** specifies that an excerpt is built using the terms that appear on both sides of the hit terms(s) */
    private static final String BOTH = "BOTH";

    /**
     * Whether we might need to trim down the excerpt to the requested size. <br>
     * Is false if this is the first time running the iterator and true otherwise.
     */
    private boolean trim;
    /** The size of half of the original desired excerpt length. Used during trimming. */
    private float origHalfSize;

    /**
     * A special term that is used to indicate we removed a candidate term because <br>
     * it was a member of the list of terms that should not be included in an excerpt.
     */
    private static final String XXXWESKIPPEDAWORDXXX = "XXXWESKIPPEDAWORDXXX";
    private static final String XXXWESKIPPEDAWORDXXX = "XXXWESKIPPEDAWORDXXX";
```